classification/model.py

```diff
@@ -43,7 +44,7 @@ def forward(self, images):
     def create_loss_fn(self,train_dataset):
         """ Returns the loss function and sets self.loss_fn
         """
-        loss_desc = self.config.loss_fnc
+        loss_desc = cfg.loss_fnc
         if loss_desc == "CE":
```

Comment:
different loss function abbreviations/settings should be in documentation and README

Reply:
agreed!

Reply:
Added abbreviation meanings to default_config.yml